README.md

```diff
@@ -186,35 +209,40 @@ This helmfiles deploys [Hashicorp's Vault](https://www.vaultproject.io/) based o
     auth:
       - type: kubernetes
         roles:
-          - name: tx-signer
-            bound_service_account_names: ["tx-signer", "vault-secrets-webhook", "vault"]
-            bound_service_account_namespaces: ["{{ .Values.vaultOperator.namespace }}", "{{ .Values.vault.namespace }}", "{{ .Values.orchestrate.namespace }}"]
-            policies: ["allow_secrets", "tx_signer_demo"]
+          - name: api-key-manage
+            bound_service_account_names: ["api-key-manage", "vault-secrets-webhook", "vault"]
+            bound_service_account_namespaces: ["{{ .Environment.Values.vaultNamespace }}", "{{ .Environment.Values.vaultOperatorNamespace }}", "{{ .Environment.Values.orchestrateNamespace }}"]
+            policies: ["allow_secrets", "api_key_manager_demo"]
+      {{ if .Environment.Values.metrics.enabled }}
+      - type: kubernetes
+        roles:
+          - name: prometheus
+            bound_service_account_names: prometheus
+            bound_service_account_namespaces: {{ .Environment.Values.observabilityNamespace }}
+            policies: prometheus
+      {{ end }}
 ```
 - PVC
 - Service Account
 - RBAC configuration
 
+Warning (Temporary): As of today the Orchestrate Vault Plugin is mounted to Hashicorp Vault with `mLock` disabled which is not recommended in production. A fix of the Vault Operator to be compatible with third-party plugins will come later. 
+
 
 As set in the existing envinonment configurations in the `environments` directory, the `tx-signer` has to be connected to the harshicorp vault with the following values:
 
-- `SECRET_STORE`: Secret storage type. Use `hashicorp` to connect use Harshicorp Vault instance.
-- `VAULT_MOUNT_POINT`: Root name of the secret engine. Value is the name of the `path` variable in `secrets` structure in Harshicorp Vault configuration.
-- `VAULT_SECRET_PATH`: Path of secret key store of ethereum wallet. Value is the `rules: path` variable in `policies` structure in Harshicorp Vault configuration.
-- `VAULT_ADDR`: Hostname and port of Harshicorp Vault instance.
-- `VAULT_CACERT`: Path to a PEM-encoded CA certificate file on the local disk. This file is used to verify the Vault server's SSL certificate.
-- `VAULT_SKIP_VERIFY`: Do not verify Vault's presented certificate before communicating with it.
+- `SECRET_STORE`: Secret storage type (default: hashicorp)
+- `VAULT_MOUNT_POINT`: Root name of the secret engine (default: orchestrate)
+- `VAULT_SECRET_PATH`: Path of secret key store of ethereum wallet (default: {{ .Environment.Values.orchestrateNamespace }}/keys)
```

Comment:
If this variable is not used any more. We are always saving keys under `/orchestrate/{ethereum}/accounts`

Reply:
removed